packages/chat/components/chat/src/chat.template.ts

```diff
@@ -0,0 +1,99 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html } from 'lit';
+import Edit16 from '@carbon/web-components/es/icons/edit/16';
+import ThumbsUp16 from '@carbon/web-components/es/icons/thumbs-up/16';
+import ThumbsDown16 from '@carbon/web-components/es/icons/thumbs-down/16';
+import ArrowRight16 from '@carbon/web-components/es/icons/arrow--right/16';
+import User24 from '@carbon/web-components/es/icons/user/24';
+import Search24 from '@carbon/web-components/es/icons/search/24';
+
+/**
+ * Lit template for test-input
+ *
+ * @param {object} customElementClass Class functionality for the custom element
+ * @returns {TemplateResult<1>} Lit html template
+ */
+export function chatTemplate(customElementClass) {
+  const { 
+    _handleInput: handleInput, 
+    _sendInput: sendInput,
+    _setMessageText: setMessageText,
+    _messages: messages,
+    _messageText: messageText,
+    _queryInProgress: queryInProgress,
+    _handleHoverIn: handleHoverIn,
+    _handleHoverOut: handleHoverOut,
+    _handleEdit: handleEdit,
+    _handleFeedback: handleFeedback,
+    _handleAPIselection: handleAPIselection 
+  } = customElementClass;
+
+    return html`<div class="chat-container">
```

Comment:
You'll need to import the following:
```
import { settings } from '@carbon/ai-utilities/es/settings/index.js';
const { stablePrefix: c4aiPrefix } = settings;
```
and you'll be able to use the prefix in the classnames ie:
```
<div class="${c4aiPrefix}--chat__container" ...
```